src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -202,6 +187,8 @@ class SidebarPanels extends React.Component<Props> {
                         )}
                     />
                 )}
+                {/* This is before the activity feed subroutes because otherwise it doesn't get
```

Comment:
Is this still needed with the addition of the `(comments|tasks)` regex below? If not, can we also move the main Activity route back to its original position?